packages/react-charting/etc/react-charting.api.md

```diff
@@ -28,6 +28,14 @@ import { TimeLocaleDefinition } from 'd3-time-format';
 // @public
 export const AreaChart: React_2.FunctionComponent<IAreaChartProps>;
 
+// @public (undocumented)
+export enum BarWidthEnum {
```

Comment:
Lets use string literal instead of enum. String literal is extensively used in slots in v9.
Enum is easier to maintain but hard on readability.
We will figure out guidance for the better approach.

Reply:
done